### DISCUSSION TITLE
stat-computing-r-python/part-wrangling/01-data-input

### DISCUSSION THREAD

**Discussion by giscus** [srvanderplas/stat-computing-r-python#9] [General]
# stat-computing-r-python/part-wrangling/01-data-input

https://srvanderplas.github.io/stat-computing-r-python/part-wrangling/01-data-input.html

**Comment by rlnayituriki** [comment]
I still can't get how you read a google sheet document in R.

**Comment by henryrscharf** [comment]
Hello, the link for footnote [4] is 404-ing for me, but I think the article is also here: https://medium.com/data-science/loading-binary-data-to-numpy-pandas-9caa03eb0672

**Reply by srvanderplas** [reply]
I can't quite manage to keep up with you, but if you want to submit a PR for the link fixes you've discovered (thanks so much!) I'd be happy to merge it in.

Sorry for my slow responses -- my department was eliminated and the classes the book was written for aren't going to be taught any longer, and I'm in "wait and see" mode to figure out whether I'm going to have the time to maintain the book at my new institution.

**Comment by henryrscharf** [comment]
Sure thing. I just submitted the PR. If I notice any other small issues
like these in the future, I'll go the PR route instead of the comments.

No need to apologize. I've been following the story about your department.
It's been terrible to watch. I feel awful for all the Nebraskans who will
be harmed by your BOR's bad decision making, and of course for all the hard
working faculty who are undeserving casualties of it all. I'm not sure if
there are any legal pathways still open for pushing back against all the
bogus changes, or how much energy remains among your colleagues for such an
effort, but if there's any hope at all for undoing all the mistakes, I'm
rooting for you. Whatever happens, I wish you luck with your new
institution.

FWIW, I am hopeful that the University of Arizona will be hiring faculty in
Statistics and Data Science next cycle (we will be replacing some recent
departures). If there is anyone from Nebraska, yourself included, who would
be interested to talk more about our own relatively nascent graduate
programs and what it's like to work here, I'm happy to chat anytime.

Best,
Henry

On Tue, Apr 14, 2026 at 7:49 AM Susan VanderPlas ***@***.***>
wrote:

> I can't quite manage to keep up with you, but if you want to submit a PR
> for the link fixes you've discovered (thanks so much!) I'd be happy to
> merge it in.
>
> Sorry for my slow responses -- my department was eliminated and the
> classes the book was written for aren't going to be taught any longer, and
> I'm in "wait and see" mode to figure out whether I'm going to have the time
> to maintain the book at my new institution.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/srvanderplas/stat-computing-r-python/discussions/9#discussioncomment-16558284>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABS5QYUGVMDXBXQKX2OIALT4VZFXFAVCNFSM6AAAAACV5EVH7WVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTMNJVHAZDQNA>
> .
> You are receiving this because you commented.Message ID:
> <srvanderplas/stat-computing-r-python/repo-discussions/9/comments/16558284
> @github.com>
>


-- 
Henry Scharf
